apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -338,7 +338,7 @@ function EventTypeSingleLayout({
             <div
               className={classNames(
                 "mt-4 rounded-md  border-gray-200 bg-white sm:mx-0 xl:mt-0",
-                disableBorder ? "border-0 xl:-mt-4 " : "p-2 md:border md:p-6"
```

Comment:
@PeerRich this seems like a mobile fix but I removed it as it broke the apps page - in the workflows page and the webhooks page fixes for this negative margin were in place I removed.